src/org/opendatakit/briefcase/ui/ExportPanel.java

```diff
@@ -88,8 +89,8 @@
   private DetailButton btnDetails;
   private JButton btnExport;
   private JButton btnCancel;
-  private JDatePickerImpl pickStartDate;
-  private JDatePickerImpl pickEndDate;
+  private DatePicker pickStartDate;
```

Comment:
Making these final lets the reader know these are not instantiated multiple times. Reduces cognitive load.
```
  private final DatePicker pickStartDate;
  private final DatePicker pickEndDate;
  private final JCheckBox checkDates;
```